www/main/workspace-bar/components/WorkspaceBar.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { remote } from 'electron';
+
+const { store } = remote.require('./app');
+
+const WorkspaceBar = () => {
+  const workspaces = ['Presenter', 'Single Screen'];
+  const defaultWsState = store.getUserPref('app.layout.presenter-view')
+    ? workspaces[0]
+    : workspaces[1];
```

Comment:
we should make this more scalability.  `app.layout.presenter-view` won't work when we have more tabs. maybe we can transition to a new UserPrefKey?

Reply:
We are changing the settings infrastructure, so will change this along with that PR.